library/cmac.c

```diff
@@ -68,6 +68,10 @@
 
 #if !defined(MBEDTLS_CMAC_ALT) || defined(MBEDTLS_SELF_TEST)
 
+#define MBEDTLS_CMAC_VALIDATE_RET(cond) \
```

Comment:
These are breaking `check_names.sh` and need to have the `MBEDTLS_` prefix removed.